src/DocumentFormat.OpenXml.Framework/Packaging/OpenXmlPackage.cs

```diff
@@ -262,7 +262,7 @@ protected override void ThrowIfObjectDisposed()
         }
 
         /// <summary>
-        /// Flushes and saves the content, closes the document, and releases all resources.
+        /// Flushes the content, closes the document, and releases all resources. If the <see cref="OpenXmlPackage.AutoSave"> property is <c>true</c>, the content will be saved before closing.
```

Comment:
```suggestion
        /// Flushes the content, closes the document, and releases all resources. If the <see cref="OpenXmlPackage.AutoSave" /> property is <c>true</c>, the content will be saved before closing.
```

whoops - looks like the xml I suggested isn't self-closing like it should be